scripts/backup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+BACKUP_PATH=$1
+
+if [ -z "$BACKUP_PATH" ];
+then
+    echo "Usage: $0 /tmp/backup-20220828.archive"
+    exit 1
+fi
+
+# Backup using mongodump excluding `trailing-trade-logs`
+mongodump --host=binance-mongo --port=27017 --gzip --archive="$BACKUP_PATH" --db=binance-bot --excludeCollection=trailing-trade-logs
```

Comment:
I think `--host`, `--port` and `--db` should be loaded from the `.env` file and set the default value if they are not changed.
Actually I face this issue because I have a different db name (`binance-bot-dev` not `binance-bot`) in my environment.
So, when we load the variables from the environment we will ensure that the values are valid.

Reply:
Ah, very good point. 

The only issue is `.env` file is not included in the Dockerfile. Of course, the reason is security. 

I can add to the `docker-compose.yml` file `environment` to be able to load env values; however, it may disrupt the traders like you who already customised it when updating the bot. You will need a manual update to the docker-compose.yml file. And if there is any change in the future, the file will be in conflict. 

Ah, I have an idea. I will pass loaded environment values as shell parameters. 

I will update it.